OxfordBodleian/Juel-Jensen/BDLaethg45.xml

```diff
@@ -51,32 +122,120 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                <desc>Mediocre but regular hand, cursive script.</desc>
+                                <seg type="ink">Black, red.</seg>
+                                <seg type="rubrication">Incipits, holy name as part of the name of the owner.</seg>
                             </handNote>
                         </handDesc>
                         
-                       <!-- <decoDesc>
-                         
+                   <!--     <decoDesc>
+                            <decoNote xml:id="d1" type="miniature" corresp="#pic1">
+                                <desc></desc>
+                                    <dimensions unit="mm">
+                           <height>286</height>
+                           <width>112</width>
+                        </dimensions>
+                            </decoNote>
+                            
+                            <decoNote xml:id="d2" type="miniature" corresp="#pic2">
+                                <desc></desc>
+                                   <dimensions unit="mm">
+                           <height>158</height>
+                           <width>112</width>
+                        </dimensions>
+                            </decoNote>
                         </decoDesc>-->
                         
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc>The name of the owner, mentioned in supplications throughout, was 
+                                        <persName ref="PRS13850WalattaS"/>. The last part of the name is spelled with variation
+                                        <foreign xml:lang="gez">ትበርህ፡</foreign>, <foreign xml:lang="gez">ትበርሂ፡</foreign> and <foreign xml:lang="gez">ትበርሄ፡</foreign>.</desc>
+                                </item>
+                                
+                                <item xml:id="e2">
+                                    <desc>There are no corrections or other scribal interventions in the text.</desc>
+                                </item>
+                                
+                                <item xml:id="e3">
+                                    <desc>In the bottom margin of the scroll, following the end of the last text, four <!--italics--> Brillenbuchstaben are 
+                                        drawn. Between them, the characters <foreign xml:lang="gez">ሽ</foreign> or  <foreign xml:lang="gez">፳</foreign>, <foreign xml:lang="gez">ሐ</foreign>,
+                                        and <foreign xml:lang="gez">ሩ</foreign> are written.</desc>
+                                </item>
+                                
+                                <item xml:id="e4">
+                                    <desc>Few remains of writing on the verso.</desc>
+                                   <!-- I cannot see them in the images, but noted this on the metadata-->
```

Comment:
I agree. I was wondering whether we should provide in some more prominent place of the description (head line where the basic codicological data is given, for instance)  the information whether the verso is empty or not (AB has in the past suggested using the opistograph/anopistograph terminology, not widespread in Ethiopian studies). We might discuss this in the meeting instead of here.